src/routes/withData.js

```diff
@@ -191,7 +191,7 @@ const withData = WrappedComponent => class WithDataComponent extends React.Compo
     const { resources } = this.props;
 
     for (const key in manifest) {
-      if (key !== 'records' && manifest[key].type === 'okapi' &&
+      if (key !== 'records' && key !== 'itemsInTransitReport' && manifest[key].type === 'okapi' &&
```

Comment:
I think it would be a bit nicer like in the suggestion below, but not necessarily. Also, since the conditions appear in two places it would to be great to have it in function, what do you think about that?

```suggestion
      if (!['records', 'itemsInTransitReport'].includes(key) && manifest[key].type === 'okapi' &&
```